RELEASING.md

```diff
@@ -0,0 +1,63 @@
+# Releasing Federation
+
+This is a quick and opinionated set of commands for building and releasing a new version of the packages in this monorepo.
+
+#### Start a release PR
+
+1. `FEDERATION_RELEASE_VERSION=X.Y.Z` (e.g. `2.0.1` or `2.0.0-beta.3` or `2.0.0-preview.99`)
+1. `git fetch origin main:main` to fetch the latest `main` branch
+1. `git checkout -b "release-$FEDERATION_RELEASE_VERSION" main`
+1. Update `CHANGELOG.md` in each repo ([more info](#changelogs))
+1. `git add ':/*CHANGELOG.md'` to add only changelogs to the commit
+1. `git commit --message "Update CHANGELOGs for v$FEDERATION_RELEASE_VERSION release"`
+1. `git push -u origin "release-$FEDERATION_RELEASE_VERSION"`
+1. `open "https://github.com/apollographql/federation/compare/main...release-$FEDERATION_RELEASE_VERSION?quick_pull=1&title=Release+$FEDERATION_RELEASE_VERSION&template=APOLLO_RELEASE_TEMPLATE.md"` to open PR in Github
+
+#### Bump version and tag release
+
+1. `npx lerna version --no-push --force-publish=\* "$FEDERATION_RELEASE_VERSION"`[^lerna-version]
```

Comment:
I'm wondering if we wanted to switch the `lerna` command (here, and in a few other places in the content) back to the `release:version-bump` command in the `package.json`, now that it's being re-introduced though?

Going closer to the bare-metal commands seems a bit backward to the spirit of whittling down to me though I understand it's uncovering true mechanisms.    And maybe this is more compatible with replacing this all entirely?

Reply:
My thoughts being new to everything: To me, it was much easier to figure out and change the process once I understood the song+dance between npm and lerna. There were a lot of things that felt like I had to uncover, and making it obvious we were using lerna was helpful.

My inclination is def based on a test size of 1, happy to change these to whatever will make deploys easiest for everyone.